src/main/java/org/gridsuite/explore/server/ExploreController.java

```diff
@@ -69,12 +70,12 @@ public ResponseEntity<Void> createStudy(@PathVariable("studyName") String studyN
     @PostMapping(value = "/explore/studies")
     @Operation(summary = "Duplicate a study")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "Study creation request delegated to study server")})
-    public ResponseEntity<Void> createStudy(@RequestParam("duplicateFrom") UUID sourceStudyUuid,
-                                            @RequestParam("studyName") String studyName,
-                                            @RequestParam("description") String description,
-                                            @RequestParam(QUERY_PARAM_PARENT_DIRECTORY_ID) UUID parentDirectoryUuid,
-                                            @RequestHeader("userId") String userId) {
-        exploreService.createStudy(sourceStudyUuid, studyName, description, userId, parentDirectoryUuid);
+    public ResponseEntity<Void> duplicateStudy(@RequestParam("duplicateFrom") UUID sourceStudyUuid,
+                                               @RequestParam("studyName") String studyName,
+                                               @RequestParam("description") String description,
+                                               @RequestParam(QUERY_PARAM_PARENT_DIRECTORY_ID) UUID parentDirectoryUuid,
+                                               @RequestHeader("userId") String userId) {
```

Comment:
remove identation changes

Reply:
The name of the method is longer, so it's normal that the formatter changed the indentation of the params no ?